### DISCUSSION TITLE
Template for SaaS without typescript ?

### DISCUSSION THREAD

**Discussion by ravikanthreddy89** [wasp-lang/wasp#1868] [Archived]
Is there a template for SAAS starter without TypeScript, haven't used TS, but wondering if there is a vanilla JSX/JS SAAS starter to work with and understand the framework.

**Comment by Martinsos** [comment]
Not at the moment unfortunately! At least not as rich as open-saas template, we are not a big enough team to maintain both at the moment.

There are other templates that come with Wasp and some of them are JS, also tutorial itself (check the docs) is JS and TS, so those should help!
